scripts/lib/zulip_tools.py

```diff
@@ -419,6 +419,12 @@ def is_root() -> bool:
         return True
     return False
 
+def run_as_root(args, **kwargs):
+    # type: (List[str], **Any) -> None
+    if not is_root():
+        args = ['sudo'] + kwargs.pop('sudo_args', []) + args
```

Comment:
We might as well add a defensive `--` here (`['sudo'] + kwargs.pop('sudo_args', []) + ['--'] + args`). This would prevent this function from being misused as in the earlier `run_as_root("-H", …)` bug.

Reply:
Also only popping `sudo_args` in the `not is_root()` case seems like a mistake.